Course/Course/Presentation/Unit/CourseUnitViewModel.swift

```diff
@@ -16,11 +16,12 @@ public enum LessonType: Equatable {
     case discussion(String, String, String)
     
     static func from(_ block: CourseBlock, streamingQuality: StreamingQuality) -> Self {
+        let mandatoryInjections: [WebviewInjection] = [.inversionCss, .ajaxCallback]
```

Comment:
The name is fine, but how about renaming it to `webviewInjections` thoughts?

Reply:
i'm not sure that it will be correct, idea is to name that variable to indicate that these are the necessary injections for all webviews.